### PR TITLE
Check spelling with Aspell

### PR DESCRIPTION
**Pull request by kemitchell**
fixes: https://github.com/npm/policies/issues/49

Following on from #27.

This PR:

1. Configures Travis CI to check spelling of all Markdown files with Aspell and the Aspell English dictionary from Ubuntu APT, failing any build that includes a spelling error.

2. Adds a "personal word list" (whitelist) of terms that Aspell should not flag.

### CODE REVIEW

**Comment by ashleygwilliams** [comment]
status of this PR? it's super old, doing some clean up rn

**Comment by ashleygwilliams** [comment]
bump, @kemitchell

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+before_script:
+  - "make"
+addons:
+  apt:
+    packages:
+      - 'aspell'
+      - 'aspell-en'
+script:
+  - "htmlproof out"
+  - '! cat *.md | aspell --dont-backup list --personal=./npm.en.pws | sort -u | sed "s/^/Not in Dictionaries: /" | grep "^"'
```
Comment (kemitchell) [inline .travis.yml]: @ashleygwilliams test scripts from Travis are here:

1. HTML checker (for hyperlinks, &c.)
2. Spell checker

**Comment by ashleygwilliams** [comment]
thoughts on rebasing and merging this @isaacs ?

**Comment by isaacs** [comment]
Done.  Thanks, @kemitchell

**Pull request closed without merging**